src/components/Func_example/Func_example.css

```diff
@@ -0,0 +1,12 @@
+
+.main {
+    position: absolute;
+    width: 713px;
+ height: 57px;
```

Comment:
use appropriate code style. I suppose that there should be space at the beginning